src/service/save-quantity.js

```diff
@@ -1,34 +1,56 @@
 import {
   postRequest,
   getRequest,
-  stockTakeURL,
+  stockOperationURL,
+  stockOperationTypeURL,
 } from '../utils/api-utils';
+import { getFormattedDate } from '../utils/date-utils';
 
 const saveEditedQuantity = async (
```

Comment:
We could rename this function as adjustQuantity since it makes an Adjustment operation on the inventory, so that it would be more readable

Reply:
Modified this from saveEditedQuantity to adjustQuantity please review